src/utilities/collection-utils.test.ts

```diff
@@ -322,6 +322,40 @@ describe("CollectionUtils", () => {
         });
     }); // end isNotEmpty
 
+    describe("#length", () => {
+        test(`when collections param is an array and has elements, it returns the length`, (): void => {
+            // Arrange
+            const array = ["test", "test"];
+
+            // Act
+            const result = CollectionUtils.length(array);
+
+            // Assert
+            expect(result).toBe(2);
+        });
+
+        test(`when collections param is null, it returns -1`, (): void => {
+            // Arrange
+            const array = null;
+
+            // Act
+            const result = CollectionUtils.length(array);
+
+            // Assert
+            expect(result).toBe(-1);
+        });
+
+        test(`when collections param is a list and has elements, it returns the length`, (): void => {
+            // Arrange
+            const list = List(["test","test"]);
+
+            // Act
+            const result = CollectionUtils.length(list);
+
+            // Assert
+            expect(result).toBe(2);
```

Comment:
Same here - reference `list.count` (or whatever the appropriate field/method is for retrieving a list's size)